<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UniPlanner</class>
 <widget class="QMainWindow" name="UniPlanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UniPlanner</string>
  </property>
  <property name="styleSheet">
   <string notr="true">alternate-background-color: rgba(255, 255, 255, 42);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Overskrift">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(139, 45, 32, 186);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700; color:#ffffff;&quot;&gt;Login p√• UniPlanner&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Mail_felt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(127, 127, 127);
gridline-color: rgb(139, 45, 32);
border-color: rgb(139, 45, 32);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhEmailCharactersOnly|Qt::ImhNoPredictiveText</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Skriv din Uni-mail her:</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Kodeord_felt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(126, 126, 126);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Skriv dit 4-cifret kodeord her:</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Login_knap">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 45, 32);
color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
font: 15pt &quot;Arial&quot;;
</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Husk_mig">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Husk mig</string>
    </property>
   </widget>
   <widget class="QLabel" name="glemt_adgangskode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;Glemt adgangskode?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="Brugernavn">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Brugernavn&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Kordeord">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Kodeord&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <zorder>splitter</zorder>
   <zorder>Mail_felt</zorder>
   <zorder>Kodeord_felt</zorder>
   <zorder>Login_knap</zorder>
   <zorder>Overskrift</zorder>
   <zorder>Husk_mig</zorder>
   <zorder>glemt_adgangskode</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
